bodhi/server/services/updates.py

```diff
@@ -467,8 +467,11 @@ def new_update(request):
     empty, the list of builds will be filled with the latest builds in this
     Koji tag. This is done by validate_from_tag() because the list of builds
     needs to be available in validate_acls().
-    Ensure that related tags ``from_tag``-pending-signing and ``from_tag``-testing
-    exists and if not create them in Koji.
+
+    If the state of the release is `pending`, ensure that related tags
```

Comment:
Composed by bodhi rather than the release state actually

Reply:
Good catch!